langgraph/pregel/__init__.py

```diff
@@ -1636,20 +1688,73 @@ def _prepare_next_tasks(
                                 else None,
                                 configurable={
                                     # deque.extend is thread-safe
-                                    CONFIG_KEY_SEND: writes.extend,
+                                    CONFIG_KEY_SEND: partial(
+                                        _local_write, writes.extend, processes, channels
+                                    ),
                                     CONFIG_KEY_READ: partial(
                                         _local_read, checkpoint, channels, writes
                                     ),
                                 },
                             ),
-                            sorted(triggers),
+                            triggers,
                         )
                     )
             else:
                 tasks.append(PregelTaskDescription(name, val))
     return checkpoint, tasks
 
 
+def _proc_input(
```

Comment:
pulling this logic out of prepare_next_tasks to share between pull and push code paths